patroni/ha.py

```diff
@@ -491,7 +526,10 @@ def reinitialize(self, cluster):
 
     def process_scheduled_action(self):
         if self.reinitialize_scheduled():
-            if self.cluster.is_unlocked():
+            if self.is_paused():
+                logger.warning('Cluster is in a pause state, can not reinitialize')
```

Comment:
What is the logic behind allowing manual failovers to the specific member and manual restarts, but not reinitializes?